packages/teams-js/src/public/tasks.ts

```diff
@@ -103,6 +112,13 @@ export namespace tasks {
     };
     return botUrldialogInfo;
   }
+
+  /**
+   * Sets the height and width of the TaskInfo object to the original height and width, if initially specified,
+   * otherwise uses the height and width values corresponding to TaskModuleDimension.Small
```

Comment:
>TaskModuleDimension.Small [](http://example.com/codeflow?start=65&length=25)

Should this somehow be a hyperlinked reference to the definition of this?

Reply:
(similar question about references to `TaskInfo` and `UrlDialogInfo` in new docs above this)

Reply:
Made the change, appreciate the feedback!